src/injected/analyzers/unified-result-sender.ts

```diff
@@ -23,15 +24,30 @@ export class UnifiedResultSender {
     ) {}
 
     public sendAutomatedChecksResults: PostResolveCallback = (axeResults: AxeAnalyzerResult) => {
-        this.sendResults(axeResults, this.convertScanResultsToUnifiedResults);
+        this.sendResults(axeResults.originalResult, this.convertScanResultsToUnifiedResults);
     };
 
     public sendNeedsReviewResults: PostResolveCallback = (axeResults: AxeAnalyzerResult) => {
-        this.sendResults(axeResults, this.convertScanResultsToNeedsReviewUnifiedResults);
+        this.sendResults(
+            this.filterNeedsReviewResults(axeResults.originalResult),
+            this.convertScanResultsToNeedsReviewUnifiedResults,
+        );
+    };
+
+    private filterNeedsReviewResults = (results: ScanResults): ScanResults => {
```

Comment:
As we discussed offline, this might be a good thing to move into a separate class as the unified result sender is increasing in complexity.